test/command/test-base.rb

```diff
@@ -122,6 +122,132 @@ def test_pretty_print
     end
   end
 
+  class CovnertToElasticsearchFormatTest < self
+    sub_test_case("non_target_command") do
```

Comment:
We don't need to use "_" as word separator in `sub_test_case`. We can use a whitespace.